ISlide.js

```diff
@@ -868,9 +855,7 @@ class ISlide extends HTMLElement {
         id: this.#slideNumber,
         scale: scale,
         defaultViewport: viewport,
-        eventBus: new pdfjsViewer.EventBus(),
-        textLayerFactory: new pdfjsViewer.DefaultTextLayerFactory(),
```

Comment:
IIRC, these were needed to make sure to produce accessible rendering of PDF - did you have a chance to check that the changes in PDFPageView constructor don't affect that particular result (e.g. that links are still clickable, text selectable)?

Reply:
I checked again today (and switched to jsdelivr.com because unpkg.com is currently down) and confirm that links are still clickable and text selectable.

I don't know if there was something else, the class no longer exists and there's no documentation about this. Now, there's a `textLayerMode` option whose default value is `ENABLE` and that triggers accessibility related code within the library. So we seem to be good.